docs/scripting/resources/gametextstyles.md

```diff
@@ -0,0 +1,82 @@
+---
+title: GameText Styles
+
+description: GameText Styles to be used in textdraws and gametext
+---
+
+To be used with [GameText](https://wiki.open.mp/docs/scripting/functions/GameTextForPlayer) and [textdraws](https://wiki.open.mp/docs/scripting/functions/TextDrawCreate).
```

Comment:
The domain has changed for the wiki, but these links should always use relative _paths_ rather than full URLs.

Since this file is in the `resources` directory, you can use `..` to go up a directory, so you can write:

```
 [GameText](../functions/GameTextForPlayer)
```

Reply:
Done